migrations/826-user-drop-source.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE users DROP COLUMN `source`;
```

Comment:
Maybe... but I wouldn't rush into it. Some of them might still be using their AMO accounts.